### PR TITLE
Fix Crashlytics Crash

### PR DESCRIPTION
**Pull request by vanniktech**
Unfortunately there's a new error now:

> w: Classpath entry points to a non-existent location: /Users/nik/tmp/OpenConference-android/app/build/tmp/kapt/playgroundDebug/classFileStubs
> :app:compilePlaygroundDebugJavaWithJavac
> warning: @AutoValue classes cannot have abstract methods other than property getters and Builder converters
> warning: @AutoValue classes cannot have abstract methods other than property getters and Builder converters
> warning: @AutoValue classes cannot have abstract methods other than property getters and Builder converters
> warning: @AutoValue classes cannot have abstract methods other than property getters and Builder converters
> Note: /Users/nik/tmp/OpenConference-android/app/build/generated/source/kapt/playgroundDebug/com/openconference/model/database/AutoValue_SessionAutoValue.java uses unchecked or unsafe operations.
> Note: Recompile with -Xlint:unchecked for details.
> 4 warnings
> :app:compilePlaygroundDebugKotlinAfterJava
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (17, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (18, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (19, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (56, 38): Unresolved reference: Timeline
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (56, 47): Unresolved reference: Tweet
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineFragment.kt: (57, 42): Unresolved reference: TweetTimelineListAdapter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (5, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (6, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (7, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (8, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (9, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (10, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (11, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (16, 25): Unresolved reference: Timeline
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (16, 34): Unresolved reference: Tweet
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (16, 43): Unresolved reference: SearchTimeline
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (25, 31): Unresolved reference: Callback
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (25, 40): Unresolved reference: TimelineResult
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (25, 55): Unresolved reference: Tweet
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (27, 7): 'success' overrides nothing
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (27, 36): Unresolved reference: Result
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (27, 43): Unresolved reference: TimelineResult
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (27, 58): Unresolved reference: Tweet
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (31, 7): 'failure' overrides nothing
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelinePresenter.kt: (31, 39): Unresolved reference: TwitterException
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineView.kt: (4, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineView.kt: (5, 12): Unresolved reference: twitter
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineView.kt: (13, 29): Unresolved reference: Timeline
> e: /Users/nik/tmp/OpenConference-android/app/src/main/java/com/openconference/twitter/TwitterTimelineView.kt: (13, 38): Unresolved reference: Tweet
> :app:compilePlaygroundDebugKotlinAfterJava FAILED
> FAILURE: Build failed with an exception. 
> - What went wrong:
>   Execution failed for task ':app:compilePlaygroundDebugKotlinAfterJava'.
>   Compilation error. See log for more details

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -35,7 +35,10 @@ android {
 
   productFlavors {
 
-    playground {} // Reserved for development
+    playground {
+      // Reserved for development
+      ext.enableCrashlytics = false
+    }
```
Comment (sockeqwe) [inline app/build.gradle:41]: what's the idea behind that?
Crashlytics should be an additional  / optional dependency every build flavor ...

**Comment by sockeqwe** [comment]
I have setup some dependency wrong:

``` groovy
droidconBerlinCompile(libs.twitterKit) 
```

should be

``` groovy
compile(libs.twitterKit) {...}
```

File: app/build.gradle (hunk 1 above)
Comment (vanniktech) [inline app/build.gradle:41]: I thought you only wanted it for droidconBerlin and hence I deactivated it for playground.

**Comment by vanniktech** [comment]
Uh makes sense. Should I change it will you or even create new   PR?

**Comment by sockeqwe** [comment]
If you don't mind, I think I will take on this issue, because I think I know how to fix it quickly

**Comment by vanniktech** [comment]
Sure go ahead :)

**Pull request closed without merging**